clients/extension/src/utils/transaction-provider/osmosis/index.ts

```
@@ -1,24 +1,23 @@
 
 export default class OsmosisTransactionProvider extends CosmosTransactionProvider {
   constructor(
     chainKey: Chain,
     chainRef: { [chainKey: string]: CoinType },
     dataEncoder: (data: Uint8Array) => Promise<string>,
-    walletCore: WalletCore,
+    walletCore: WalletCore
   ) {
-    super(chainKey, chainRef, dataEncoder, walletCore);
+    super(chainKey, chainRef, dataEncoder, walletCore)
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)